src/modules/new-request-form/fields/LookupField.tsx

```diff
@@ -114,14 +113,20 @@ export function LookupField({
 
         const data = await response.json();
         if (response.ok) {
-          setOptions(
-            data.custom_object_records.map(
-              ({ name, id }: { name: string; id: string }) => ({
-                name,
-                value: id,
-              })
-            )
+          let fetchedOptions = data.custom_object_records.map(
+            ({ name, id }: { name: string; id: string }) => ({
+              name,
+              value: id,
+            })
           );
+          if (selectedOption) {
+            fetchedOptions = fetchedOptions.filter(
+              (option: FieldOption) => option.value !== selectedOption.value
+            );
+            fetchedOptions = [selectedOption, ...fetchedOptions];
```

Comment:
Is this to guarantee that the selected option appears at the top?

Reply:
I think so 😬 just manually setting the order

Reply:
Is this the preferred experience for when an option is already selected?
I was going through the JIRA ticket and the slack thread but couldn't see it mentioned there.

Reply:
It is the same experience as in Lookup Field in Support and we want to make this fields almost the same.

Reply:
Got it! 👍🏼